src/main/resources/db/changelog/changes/09-create-shopping-carts-table.yml 

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-carts-table
+      author: Roma1990Melnichuk
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: user_id
```

Comment:
remove
Add soft delete approach to ShoppingCart entity

Reply:
Not fixed

Reply:
remove adding column `name: user_id`